aspnetcore/index.md

```diff
@@ -17,7 +17,7 @@ By [Daniel Roth](https://github.com/danroth27), [Rick Anderson](https://twitter.
 
 ASP.NET Core is a cross-platform, high-performance, [open-source](https://github.com/aspnet/home) framework for building modern, cloud-based, Internet-connected applications. With ASP.NET Core, you can:
 
-* Build web apps and services, IoT apps, and mobile backends.
+* Build web apps and services, [IoT](https://wikipedia.org/wiki/Internet_of_things) apps, and mobile backends.
```

Comment:
Should we link to our own property instead, to help drive traffic there? If so, I'd suggest https://www.microsoft.com/internet-of-things/.